qhana_plugin_registry/api/models/templates.py

```diff
@@ -60,39 +60,65 @@ class TemplateTabSchema(ApiObjectSchema):
     plugins = ma.fields.Nested(ApiLinkSchema)
 
     @staticmethod
-    def validate_filter(filter_dict: dict):
+    def validate_filter(filter_dict: dict, path: list[str] = []):
```

Comment:
do not use mutable default values for parameters! (they are only evaluated once when the function is defined)
```suggestion
    def validate_filter(filter_dict: dict, path: tuple[str, ...] = ()):
```
or
```suggestion
    def validate_filter(filter_dict: dict, path: str = ""):
```